robot/basestation/static/js/elements/feed/navbar.js

```diff
@@ -134,22 +134,22 @@ $(document).ready(() => {
         buttonText.includes('Rover') &&
         $('#toggle-rover-listener-btn')[0].checked == true
       ) {
-        // sendRoverCommand(cmd) // rover commands not yet implemented
+        // sendRoverCommand(command) // rover commands not yet implemented
```

Comment:
why are rover, science and PDS commands disabled? this sort of defeats the purpose of the feature, to make sending messages to any module easier for testing/debugging.

Reply:
Oops, fixed